catalog/MDCCatalog/MDCCatalogCollectionViewCell.swift

```diff
@@ -20,22 +20,25 @@ import MaterialComponents.MaterialTypography
 
 class MDCCatalogCollectionViewCell: UICollectionViewCell {
 
-  var label = UILabel()
-  let pad = CGFloat(14)
-  let tile = MDCCatalogTileView(frame: CGRect.zero)
+  fileprivate struct Constants {
+    static let imageWidthHeight: CGFloat = 80
+    static let xPadding: CGFloat = 14
+    static let yPadding: CGFloat = 16
+  }
+
+  private lazy var label = UILabel()
```

Comment:
Why not have this match the other lazy properties by setting the font and textColor in the declaration?

Reply:
Done